python-sdk/tests/test_example_dags.py

```diff
@@ -89,7 +90,11 @@ def order(dag_id: str) -> int:
 @pytest.mark.parametrize("dag_id", sorted(dag_bag.dag_ids, key=order))
 def test_example_dag(session, dag_id: str):
     dag = dag_bag.get_dag(dag_id)
-    wrapper_run_dag(dag)
+    dag_run = wrapper_run_dag(dag)
+    print(dag_run)
+    for ti in dag_run.get_task_instances():
+        if ti.state == "failed":
+            assert False
```

Comment:
This doesn't look right, 🤔 

Did we confirm that not all example DAGs were running?

Reply:
Yeah i am trying to do that only. This check is for if dag has run but has task failure check. 
i am yet to add a check for all dags ran or not.